app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -235,13 +247,48 @@ private JsonObject customContentTab(String tabType, String name, JsonObject prop
         return result;
     }
 
-    // Smart-Seq-like experiments will contain the substring “smart” in their technology types
-    private static boolean isSmartExperiment(Collection<String> technologyType) {
-        return technologyType.stream()
-                .anyMatch(type -> type.toLowerCase().matches("smart" + "-(?:.*)"));
+    public ImmutableMap fetchDefaultPlotMethodAndParameterisation(String experimentAccession) {
+        return cellPlotService.fetchDefaultPlotMethodWithParameterisation(experimentAccession);
     }
 
-    public ImmutableMap fetchDefaultPlotMethodAndParameterisation(String experimentAccession){
-        return cellPlotService.fetchDefaultPlotMethodWithParameterisation(experimentAccession);
+    public JsonArray getMarkerGeneMetadata(JsonArray metadata, String experimentAccession) {
+        JsonArray markerGenesArray = new JsonArray();
+
+        metadata.forEach(item -> {
+            JsonObject jsonObject = item.getAsJsonObject();
+            processMarkerGene(jsonObject, experimentAccession,
+                    "inferred_cell_type_-_ontology_labels",
```

Comment:
You can extract this to a constant or a local variable as it fits.
If you use this other places inside the class: global constant, otherwise local variable.